README.md

```diff
@@ -62,6 +63,13 @@ if you're on campus use the `ubcsecure` or `resnet` networks for best results.
 - All files **must** be formatted properly. Formatting will be enforced with the `clang-format` tool. 
     - To check and fix formatting, from the `Snowflake` folder run `./clang_format/fix_formatting.sh BRANCH_NAME`, where `BRANCH_NAME` is the name of the branch you intend to merge your code into (ex. `iarrc` or `core`). This script will fix any improperly formatted code, but will refuse to change any files with uncommited changes (to prevent you losing work)
 
+### Pull Requests
+Below are some notes we would like to highlight about pull requests: 
+- Once your pull request has been approved, please proceed to merge and close the pull request yourself
+- When your pull requests receive comments, please reply to each comment individually. 
+- Once your branch has been merged, please delete the branch. 
```

Comment:
This one isn't really required, since people only have branches on their own forks.